iss.py

```diff
@@ -1,11 +1,101 @@
 #!/usr/bin/env python
 
-__author__ = '???'
+__author__ = 'katran009 and google'
 
+import json
+import urllib.request
+import turtle
+import time
 
-def main():
-    pass
+# API addresses
+ASTRONAUTS_URL = 'http://api.open-notify.org/astros.json'
+LOCATION_URL = 'http://api.open-notify.org/iss-now.json'
 
+# Image paths
+BACKGROUND_PATH = 'images/map.gif'
+ISS_PATH = 'images/iss.gif'
 
-if __name__ == '__main__':
-    main()
+
+ISS = turtle.Turtle()
+WORLD = turtle.Screen()
+
+
+def api_data(url):
+    """
+    Get the JSON data from the specified NASA API.
+    """
+    url = url
+    response = urllib.request.urlopen(url)
+    # data came as bytes so convert into string to be parsed
+    string_response = response.read().decode("utf-8")
+    result = json.loads(string_response)
```

Comment:
Not sure what 'url = url' does here.
This becomes much simpler with `requests` .. I encourage you to give it a try you won't regret it